nl_server/routes.py

```diff
@@ -48,10 +48,12 @@ def healthz():
   if default_embeddings_loaded:
     return 'OK', 200
 
-  default_index_type = current_app.config[config.ENV_KEY].default_indexes[0]
-  if not default_index_type:
-    logging.warning('Health Check Failed: Default index name empty!')
+  nl_env = current_app.config[config.ENV_KEY]
+  if not nl_env.default_indexes:
+    logging.error('Health Check Failed: Default index name empty!')
```

Comment:
There are warnings below, so change them all or not?

(original rational for warning was I thought this can happen while booting up, i think i saw this crash in custom dc - https://github.com/datacommonsorg/website/pull/4240)

Reply:
Oh. Let me update here in next PR. I hope we don't need to make a real prediction call in /healthz then all logic here can be simplified.

Reply:
ah, makes sense!